### ISSUE TITLE
server undefined vars

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#2014]
Possibly undefined vars in server code, found with pylint.

```
auth/__init__.py:542:21: E0602: Undefined variable '_COOKIE_LOGIN_SESSION' (undefined-variable)
job_api.py:98:11: E0602: Undefined variable 'wsgiref' (undefined-variable)
comsol_register.py:22:11: E0602: Undefined variable 'server' (undefined-variable)
runner/docker.py:194:60: E0602: Undefined variable 'cmd' (undefined-variable)
runner/docker.py:245:56: E0602: Undefined variable '_tls_d' (undefined-variable)
runner/docker.py:627:61: E0602: Undefined variable 'hosts' (undefined-variable)
runner/background.py:120:12: E0602: Undefined variable 'reraise' (undefined-variable)
job_driver/__init__.py:47:8: E0602: Undefined variable 'self' (undefined-variable)
job_driver/local.py:35:79: E0602: Undefined variable 'selfId' (undefined-variable)
job_driver/local.py:44:25: E0602: Undefined variable 'kind' (undefined-variable)
http_reply.py:227:13: E0602: Undefined variable 'pkconfig' (undefined-variable)
```

**Comment by robnagler** [comment]
- auth/__init__.py:542:21: E0602: Undefined variable '_COOKIE_LOGIN_SESSION' (undefined-variable) d738a06fe2b7b4fe993f5608d193525cf9203228
- runner/background.py:120:12: E0602: Undefined variable 'reraise' (undefined-variable) 169a2b46a8d655b24ab64557e6faa412b5505c67
- http_reply.py:227:13: E0602: Undefined variable 'pkconfig' (undefined-variable) bac60f53396c123fdf92fa326a89418322beace2
- comsol_register.py:22:11: E0602: Undefined variable 'server' (undefined-variable) e4f20bf28f02f0b691790d3432dd7657689d2ea8

Unused module (leaving around for a bit):
- runner/docker.py:194:60: E0602: Undefined variable 'cmd' (undefined-variable)
- runner/docker.py:245:56: E0602: Undefined variable '_tls_d' (undefined-variable)
- runner/docker.py:627:61: E0602: Undefined variable 'hosts' (undefined-variable)

Under construction:
- job_api.py:98:11: E0602: Undefined variable 'wsgiref' (undefined-variable)
- job_driver/__init__.py:47:8: E0602: Undefined variable 'self' (undefined-variable)
- job_driver/local.py:35:79: E0602: Undefined variable 'selfId' (undefined-variable)
- job_driver/local.py:44:25: E0602: Undefined variable 'kind' (undefined-variable)

**Comment by robnagler** [comment]
Fix dd9b0e8
